app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+  before_action :require_login
```

Comment:
Histoire de rester consistant avec les librairies standard, je renommerai le `require_login` en `authenticate_user!`

Cf https://github.com/heartcombo/devise#controller-filters-and-helpers

Reply:
Ok !